qdrant-landing/content/documentation/qdrant-alternatives.md

```diff
@@ -0,0 +1,78 @@
+|----------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Index**      | **Collection** | Pinecone's index is an organizational unit for storing and managing vectors of the same size. The index is tightly coupled with hardware (pods). Qdrant uses the collection to describe a similar concept, however, a single instance may handle multiple collections at once. |
+| **Collection** | N/A            | A collection in Pinecone is a static copy of an *index* that you cannot query, mostly used as some sort of backup. There is no such concept in Qdrant, but if you want to back your collection up, you may always create a snapshot.                                           |
+| **Namespace**  | N/A            | Namespaces allow the partitioning of the vectors in an index into subsets. Qdrant does not have such a concept, as you can always create multiple collections.                                                                                                                 |
+| **Metadata**   | **Payload**    | Additional attributes describing a particular object, other than the embedding vector. Both engines support various data types, but Pinecone metadata is key-value, while Qdrant supports any JSON-like objects.                                                               |
+| **Query**      | **Search**     | Name of the method used to find the nearest neighbors for a given vector, possibly with some additional filters applied on top.                                                                                                                                                |
+| N/A            | **Scroll**     | Pinecone does not offer a way to iterate through all the vectors in a particular index. Qdrant has a `scroll` method to get them all without using search.                                                                                                                     |
+
+## Known limitations
+
+1. Pinecone does not support arbitrary JSON metadata, but a flat structure with strings, numbers, booleans, or lists of strings used as values. Qdrant accepts any JSON object as a payload, even nested structures.
+2. NULL values are not supported in Pinecone metadata but are handled properly by Qdrant.
+3. The maximum size of Pinecone metadata is 40kb per vector. 
+4. Pinecone, unlike Qdrant, does not support geolocation and filtering based on geographical criteria.
+5. Qdrant allows storing multiple vectors per point, and those might be of a different dimensionality. Pinecone doesn't support anything similar.
+6. Vectors in Pinecone are mandatory for each point. Qdrant supports optional vectors.
```

Comment:
Should probably elaborate why `optional` vectors are useful? Add a one line use case example? This is not a common term or support